_episodes/first_example_recipe.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Running a recipe (First example)"
+teaching: 20
+exercises: 25
+questions:
+- "What is a recipe?"
+- "How can I do the same preprocessing on many different datasets?"
+- "What are the files and directories after running a recipe?"
+- "What happens when I run a recipe?"
+objectives:
+- "Run an ESMValTool recipe"
+- "Understand the purpose of different settings in the recipe"
+- "Inspect the output directories"
+- "Examine the log information"
+keypoints:
+- "A recipe does not break by fiddling with it"
+- "Log information is useful when interpreting the first warnings/errors"
+- "The dataset section in the recipe relates to Understanding the directory structure of CMIP data on your server/disk and knowing how to use data from different experiments such as CMIP/ScenarioMIP"
+---
+
+This episode describes how ESMValTool recipes work, how to run a recipe and how to explore the recipe output. By the end of this episode, you should be able to run your first recipe, look at the recipe output, modify a recipe, explore and run some basic recipe debugging.
+
+## Introduction to Recipes
+
+Recipes are the instructions that you give to ESMValTool that tell it what you want to do. This includes four main sections: datasets, preprocessors, diagnostics and description.
+
+- datasets: what datasets you want to use, including
+ - the time range and time resolution,
+ - the MIP, ensemble member,
+ - the experiment (i.e. historical, ssp125 etc...),
+ - and the grid type (CMIP6 only)
+
+ This section can also be optional, as datasets can no preprocessing is needed.
+
+- preprocessors: general operations applied to a dataset before handling it in a diagnostic, listing
+ - which preprocessor modules to apply,
+ - the order to apply them,
+ - and the preprocessor arguments.
+
+ This section can also be optional, if no preprocessing is needed.
+
+- diagnostics: all the information about the diagnostic, including
+ - list of variables to evaluate (with their respective configurations),
+ - the desired diagnostic script to use,
+ - and additional diagnostic script options or arguments, if needed.
+
+ Also Include additional datasets beyond those included in the datasets section mentioned above, for instance variable specific observational data.
+
+- description: a brief description of the recipe, including
+ - who wrote the recipe and who maintains it,
+ - which project is responsible for it,
+ - and which publications, references are linked with the recipe.
+
+ Note that the authors, publications and references are to be named in the config-references.yml
+
+This information ...
+
+## How to run ESMValTool
+
+Once you’ve set up your conda environment and installed ESMValTool (see episode #2 LINK) and set up your config-user.yml file to correctly match you local environment, (see episode #3 LINK), ESMValTool is invoked using a simple command:
```

Comment:
How do we add a link to the other episodes here?

Reply:
https://carpentries.github.io/lesson-example/04-formatting/index.html